ipcsdk/gen/gen.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	ipcsdk "github.com/consensus-shipyard/go-ipc-types/ipcsdk"
+	gen "github.com/whyrusleeping/cbor-gen"
+)
+
+func main() {
+	if err := gen.WriteTupleEncodersToFile("./cbor_gen.go", "ipcsdk",
+		ipcsdk.IPCAddress{},
```

Comment:
As I mentioned above, these do not look good from a naming perspective.

Reply:
Agree, I'll probably stick with `sdk.IPCAddress` incorporating @ranchalp's feedback.